docs/src/main/asciidoc/_javaee-guide/Jakarta_RESTful_Web_Services.adoc

```diff
@@ -9,14 +9,14 @@ ifdef::env-github[:imagesdir: ../images/]
 [[tutorial-overview]]
 == Tutorial Overview
 
-This chapter describes the Java API for RESTful web services (JAX-RS,
-defined in http://jcp.org/en/jsr/detail?id=311[JSR331]).
+This chapter describes the Jakarta RESTful Web Services
+(defined in http://jcp.org/en/jsr/detail?id=311[JSR331]).
```

Comment:
That URL points to an older Oracle spec document. Version 2.1.6, currently implemented by RESTEasy, is here: https://jakarta.ee/specifications/restful-ws/2.1/. Unfortunately, the HTML and PDF versions are currently broken, but I've sent email about it.